assets/performanceprofile/scripts/add-offline-cpus.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/bash
```

Comment:
more than _ADD_ offline cpus, this is more like _SET_ offline cpus (or even better set cpus offline)

Reply:
done